scandium-core/pom.xml

```diff
@@ -61,8 +64,8 @@
 						Running the tests in parallel may cause problems with rebinding to a an address within a test,
 						e.g. if another test running in parallel binds to that address in between.
 					-->
-					<parallel>classes</parallel>
-					<threadCount>8</threadCount>
+					<!-- parallel>classes</parallel>
+					<threadCount>8</threadCount-->
```

Comment:
Is it normal that parallel test was deactivated ? this seems not linked to this PR ?

Reply:
OK, I have to check this. It was necessary for the "repeating runner", because parallel testing seems to use also a own runner. But I didn't check it for the "rule" only.